lib/mongo/server/app_metadata.rb

```diff
@@ -109,25 +109,31 @@ def initialize(options = {})
       # @api private
       attr_reader :purpose
 
+      # @return [ Hash | nil ] Information server api.
```

Comment:
What do you mean by "information server api"? I think this hash is referred to elsewhere as "server API parameters".

Reply:
Corrected.